frontend/containers/breadcrumbs/component.tsx

```diff
@@ -0,0 +1,128 @@
+import { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { Paths } from 'enums';
+
+import type { BreadcrumbsProps } from './types';
+
+export const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  className,
+  substitutions: propSubstitutions = {},
+}: BreadcrumbsProps) => {
+  const intl = useIntl();
+  const { route, asPath } = useRouter();
+
+  // Predefined substitutions for this project. Ideally this would live in a constants file,
+  // but due to the need to use `react-intl` they're kept here.
```

Comment:
I found the same issue on another part of the code. I still moved the contants to a different files (`helpers.ts`) and passed `intl` as a prop. But it's ok to keep it here.